charts/spire/charts/spire-server/values.yaml

```diff
@@ -685,6 +685,29 @@ nodeAttestor:
     enabled: true
     ## @param nodeAttestor.k8sPsat.serviceAccountAllowList [array] Allowed service accounts for Psat nodeattestor
     serviceAccountAllowList: []
+    ## @param nodeAttestor.k8sPsat.audience [array] Audience for token validation. If it is set to an empty array ([]), Kubernetes API server audience is used
+    audience: ["spire-server"]
+    ## @param nodeAttestor.k8sPsat.allowedNodeLabelKeys [array] Node label keys considered for selectors
+    allowedNodeLabelKeys: []
+    ## @param nodeAttestor.k8sPsat.allowedPodLabelKeys [array] Pod label keys considered for selectors
+    allowedPodLabelKeys: []
+  externalK8sPsat:
+    ## @param nodeAttestor.externalK8sPsat.enabled Enable Psat k8s nodeattestor for external Kubernetes clusters
+    enabled: false
+    defaults:
```

Comment:
Why the extra `defaults` level?

Reply:
Also, the defaults for the internal cluster can be different from external ones. For example, the serviceAccountAllowList will always need to be different in the external case.

Reply:
Does the doc tool allow to define a doc at the defaults level? Maybe we could add the explanation there to make it clear. Not sure if that will actually render in the md file though.

Reply:
Yeah, I don't think we can get it into the md file nicely. Its a case we can document really well in the spiffe.io docs though since we have more control over it, so maybe thats enough?'

Reply:
Is this because its an array and we cant set defaults for arrays?